network-api/networkapi/templates/pages/buyersguide/article_page.html

```diff
@@ -46,64 +46,51 @@ <h1 class="tw-h1-heading">{{ page.title }}</h1>
   </div>
 
   {% for block in page.body %}
-      {% include_block block with parent_page=page page_type="blog" %}
+    {% include_block block with parent_page=page page_type="blog" %}
   {% endfor %}
 
-  {% with related_articles=page.get_primary_related_articles  %}
-    {% if related_articles %}
-    <div class="tw-container tw-my-4">
-      <div class="tw-row">
-        <div id="article-primary-related-articles" class="tw-px-4 tw-w-8/12 tw-mx-auto">
-          {% include "fragments/buyersguide/related_reading.html" with articles=related_articles %}
+  {% with author_profiles=page.get_author_profiles  %}
+    {% if author_profiles %}
+      <div class="tw-container tw-mt-[3rem]">
```

Comment:
```suggestion
      <div class="tw-container tw-mt-7">
```

Reply:
**Suggestion:** `tw-mt-7` equals 3rem and removes the need for a custom tw-class

Reply:
Great catch! Thanks @danielfmiranda